### ISSUE TITLE
SSLError: HTTPSConnectionPool(host='en.wikipedia.org', port=443): Max retries exceeded with url: /w/api.php...

### ISSUE THREAD

**Issue by danielunderwood** [lidarr/LidarrAPI.Metadata#28]
https://sentry.io/lidarr/metadata-api/issues/436386164/

```
SSLError: HTTPSConnectionPool(host='en.wikipedia.org', port=443): Max retries exceeded with url: /w/api.php?inprop=url&redirects=&format=json&ppprop=disambiguation&prop=info%7Cpageprops&titles=Alesha_Dixon&action=query (Caused by SSLError(SSLEOFError(8, u'EOF occurred in violation of protocol (_ssl.c:590)'),))
(19 additional frame(s) were not displayed)
...
  File "lidarrmetadata/api.py", line 105, in get_artist_info
    wikipedia_links[0]['target'])

SSLError: HTTPSConnectionPool(host='en.wikipedia.org', port=443): Max retries exceeded with url: /w/api.php?inprop=url&redirects=&format=json&ppprop=disambiguation&prop=info%7Cpageprops&titles=Alesha_Dixon&action=query (Caused by SSLError(SSLEOFError(8, u'EOF occurred in violation of protocol (_ssl.c:590)'),))
```

**Comment by danielunderwood** [comment]
Fixed by #54.

**Comment by Trianamonikaa** [comment]
maybe you must remove the link that have ":", because, it make be render to the web that cannot access with your browser, 

`':' not in link['href']`
you can add that into your code

example `if re.search('/wiki/',str(link['href'])) and ':' not in link['href']`, i use that code when i want to find the external link in wikipedia
